### ISSUE TITLE
implement to_bytes method

### ISSUE THREAD

**Issue by yoshuawuyts** [datrs/sparse-bitfield#2]
From https://github.com/mafintosh/sparse-bitfield#var-buffer--bitstobuffer

**Comment by yoshuawuyts** [comment]
```rust
extern crate sparse_bitfield;

use sparse_bitfield::Bitfield;

let mut bits = Bitfield::new(1024);

let buf = Vec::with_capacity(bits.byte_len());
bits.to_bytes(&buf);
```

**Comment by yoshuawuyts** [comment]
Or like, create an iterator.

**Comment by yoshuawuyts** [comment]
Solved in #7, we implemented an iterator instead - more efficient.
